### ISSUE TITLE
Pretty large issue with support for PLA, thats been an issue since 1.8.0.57 beta

### ISSUE THREAD

**Issue by CallmeBigfat** [bambulab/BambuStudio#3945]
### Bambu Studio Version

1.9.1.56

### Where is the application from?

Bambu Lab Official website, Bambu Lab github releases

### OS version

Windows 11

### Additional system information

RTX 3080
32GB RAM
Intel 14TH Gen i9

### Printer

X1C,P1S,A1MINI

### How to reproduce

1. Add something that is 3 multi colored (plus support pla) on an angle
2. Select Sup.PLA as interface later
3. Set Top Z distance to 0.08 or lower
4. Slice and view where model meets the interface later it adds the wrong color ABOVE the interface support which shows up in prints.

### Actual results

The sliced file will print the wrong color on top of the support layer above it, the wrong color is merged into the support layer.  On orca slicer this is not an issue (even now). I do not want 0 support layer gap as my prints are complex on the bottom so 0.055-0.06 works perfect for me if i go any larger the print looks ugly on the bottom cause the gap is too large, if I do 0.00 then the support takes forever to remove about 15-20 minutes per object which is not effective at all.

![image](https://github.com/bambulab/BambuStudio/assets/72172156/2a3c0137-3372-4754-8882-650b20e9ea6e)

Older photo of Studio version 1.7.7.89 where things worked fine compared to where they broke. (Version 1.8.0.57)
![image](https://github.com/bambulab/BambuStudio/assets/72172156/ea083241-9e17-4ce2-b3a2-72a6fb0310c4)

On the left is an older beta when it happened, the right is the last time things worked perfectly (I have sinced been forced to move to Orca slicer but would like to return to Bambu Studio but cant until this is fixed) I have reached out to support but it seems they assume this is "normal" but should be fixed.

Here is a photo of the current beta and newest version of Orca Slicer you can see the difference.

![image](https://github.com/bambulab/BambuStudio/assets/72172156/6d41339b-9e30-4048-a6e5-f90341ea78fe)

### Expected results

What should happen is what happens in **Studio version 1.7.7.89**, this is when everything worked perfectly.

### Project file & Debug log uploads

[debug_network_Wed_Apr_24_02_36_16.log.zip](https://github.com/bambulab/BambuStudio/files/15090941/debug_network_Wed_Apr_24_02_36_16.log.zip)

### Checklist of files to include

- [X] Log file
- [x] Project file

**Comment by CallmeBigfat** [comment]
Issue still present in public 1.9.1.66 . Tried possibly disabling flushing into infill with still no success. 
![image](https://github.com/bambulab/BambuStudio/assets/72172156/c3cc9038-cc51-4ccd-b79e-7ce6e5547a04)

**Comment by tangketan** [comment]
It's a quite old bug. We don't have any solution yet. Will keep on tracking the issue.

**Comment by CallmeBigfat** [comment]
Thank you. I would love to migrate my workflow back to Bambu studio if this ever gets fixed.

**Comment by CallmeBigfat** [comment]
Still present in the new beta. Almost coming up on 2 years where I have not been able to use bambu studio. Would love to eventually.
![image](https://github.com/user-attachments/assets/53111911-6595-40cf-9108-8326793eabf0)

**Comment by CallmeBigfat** [comment]
Huge bummer , would LOVE to go back to bambu studio after all these years. But issue still persists. 
![image](https://github.com/user-attachments/assets/555a6564-5a42-43db-a683-0ec3fc2f16a7)

**Comment by CallmeBigfat** [comment]
2025 update , bug is still present (crazy long time) and NOW with the new "authorization" beta update Orca Slicer will not be able to utilize some functions like sending prints. This is a huge blow. I wish this bug was fixed before things get more closed garden.

**Comment by CallmeBigfat** [comment]
Still an issue 3/26/25 , version 2.0.0.95, a TRUE bummer, I waited years for a bug that cant be fixed. I will not be buying the H2D and plan to switch printer brands. I just can't wait around any longer and now the orca slicer hassle. I have been pushed our from bambu.

![Image](https://github.com/user-attachments/assets/bb8011b9-8d01-45cf-9810-ceb46a8f0b97)

**Comment by CallmeBigfat** [comment]
> It's a quite old bug. We don't have any solution yet. Will keep on tracking the issue.

Any update, my business is hurting cause I can't use bambu slicer and now have to go through the hassle of using orca slicer with the quickly baked bambu connect app. I did not update any machines yet thankfully, but would LOVE to get any feedback since its been years.

**Comment by CallmeBigfat** [comment]
V 2.0.1.50 still has the issue. Any number below 0.1mm on top Z distance just breaks everything. (Been years) Hope this post gets some tracking on reddit and the facebooks.

![Image](https://github.com/user-attachments/assets/dd65f02d-24dd-4186-9a47-441e65dbffa9)

**Comment by CallmeBigfat** [comment]
![Image](https://github.com/user-attachments/assets/d2c15092-3901-4551-b396-3c6d0a0c33db)

Still present in 2.0.2.57 :(

**Comment by CallmeBigfat** [comment]
I can upload one of the files that have the issue if you need it for better assessment. Please let me know thank you.  @tangketan  I would love to buy the H2D. But not until its fixed.

**Comment by CallmeBigfat** [comment]
STILL an issue in 2.1.0.59 , a huge bummer and utterly disappointing. 

![Image](https://github.com/user-attachments/assets/71b82cbd-8032-4d1f-8b11-4e89aafd4647)

**Comment by CallmeBigfat** [comment]
Still an issue on V 2.1.1.52 , losing all hope. @tangketan 

![Image](https://github.com/user-attachments/assets/3b2770bb-0155-406e-96e5-52ca215041eb)

**Comment by CallmeBigfat** [comment]
Still an issue on beta 2.2.0.60, bambu should be embarrassed. It's been too long I am selling the rest of my machines.

**Comment by CallmeBigfat** [comment]
still an issue after the H2C release. Disappointing. And STILL waiting for a fix.
